netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -247,6 +249,18 @@ protected NettyServer buildTransportServer() {
   }
 
   @Override
+  public NettyServerBuilder useTransportSecurity(TrustManagerFactory trustManagerFactory,
+                                                 KeyManagerFactory keyManagerFactory) {
+    try {
+      sslContext = GrpcSslContexts.forServer(trustManagerFactory, keyManagerFactory).build();
+    } catch (SSLException e) {
+      // This should likely be some other, easier to catch exception.
```

Comment:
Sorry. I guess the `build` method is still spec'ed to throw, even though it shouldn't actually ever throw when given the `TrustManagerFactory` and `KeyManagerFactory`. So I was wrong about not needing to catch and re-throw...